lib_nn/src/c/vpu_sim.c

```diff
@@ -129,81 +128,81 @@ void VSTRPV(const xs3_vpu* vpu, void* addr, unsigned mask) {
   }
 }
 
-void VLMACC(xs3_vpu* vpu, const void* addr) {
+void VLMACC(xs3_vpu *vpu, const void *addr) {
   assert_word_aligned(addr);
   if (vpu->mode == MODE_S8) {
-    const int8_t* addr8 = (const int8_t*)addr;
+    const int8_t *addr8 = (const int8_t *)addr;
 
     for (int i = 0; i < VPU_INT8_VLMACC_ELMS; i++) {
-      int64_t acc = get_accumulator(vpu, i);
+      int64_t acc = GetAccuumulator(vpu, i);
```

Comment:
There's a typo, GetAccuumulator should be GetAccumulator instead.

Reply:
ha, good spot, thanks